xml/System.Windows.Forms/ComboBox.xml

```diff
@@ -3028,11 +3028,21 @@
         <Parameter Name="keyData" Type="System.Windows.Forms.Keys" Index="1" FrameworkAlternate="netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="msg">To be added.</param>
-        <param name="keyData">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="msg">A <see cref="T:System.Windows.Forms.Message" />, passed by reference, that represents the window message to process.</param>
```

Comment:
The MS Docs website wil show the parameter's type next to the name, so mentioning a cref to the type in the comment is not necessary. It's better to use natural language. Something like this (but feel free to reword it if you want):

```suggestion
        <param name="msg">The window message to process, passed by reference.</param>
```